src/CheckoutSdk/Sessions/CreateSessionOkResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Checkout.Sessions
+{
+    public class CreateSessionOkResponse : GetSessionResponse
+    {
```

Comment:
This class is empty, is necessary?

Reply:
That is a really good question.
I based my code on Java Project. Perhaps @rsaestrela  or @Armando-Ibarra can give us more info.

Reply:
Yeah looks awful, but the SessionResponse contains the response for a OK or an ACCEPTED response, so we are returning nothing when an OK occurs.